database/tbcd/level/level_test.go

```diff
@@ -112,3 +125,265 @@ func TestMD(t *testing.T) {
 		t.Fatal("expected no return value")
 	}
 }
+
+func mergeKssMaps(source ...*map[chainhash.Hash]tbcd.Keystone) map[chainhash.Hash]tbcd.Keystone {
+	final := make(map[chainhash.Hash]tbcd.Keystone)
+
+	for _, mp := range source {
+		maps.Copy(final, *mp)
+	}
+
+	return final
+}
+
+func TestKeystoneUpdate(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.TODO())
+	defer cancel()
+
+	home := t.TempDir()
+	t.Logf("temp: %v", home)
+
+	cfg := level.NewConfig(home, "", "")
+	db, err := level.New(ctx, cfg)
```

Comment:
let's do this before each test, in the `t.Run(...)` function.  this way we start with a _fresh database_ on every test, so we know the state of the test when it starts.  the way the test is written now, each test case will leave data behind for the next case, so we're not starting with a clean state.